shared-bindings/ota/__init__.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 microDev
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#include "shared-bindings/ota/__init__.h"
+
+//| """OTA Module
+//|
+//| The `ota` module implements over-the-air update.
```

Comment:
`dualbank` no doubt better explains the general module functionality but I find `OTA` to be a more widely used term.

Reply:
I'm not a fan of `OTA` because it doesn't mean anything to newcomers `overtheair` would be clearer. I think `dualbank` is best because it could be used more broadly. @dhalbert What do you think?

Reply:
OTA to me implies there is some wireless update capability, but the wireless part is not actually in this API. In fact, it could done over USB, for instance. `dualbank` describes a particular technique, though it's also some technical jargon. How about 'live_update`? That is the functional description: an update can be done while the current firmware is running.

Reply:
so many name ideas 👍... let me add one more use case of this module... that being `dual-boot` functionality.

This module can be used to switch between:
- different versions of circuitpython
- different platforms like arduino, circuitpython, idf etc.

Reply:
I'd say `dualbank` is a better name if you want to support dual boot.